### ISSUE TITLE
Marketplace: Can you upgrdae the extension?

### ISSUE THREAD

**Issue by mikkelhegn** [spinkube/azure#27]
Not sure if we've tested yet, it it's possible to upgrade the extension. I have indications from a mail-thread that we may not be able to.

**Comment by Mossaka** [comment]
Yes you can upgrade the extension. You can even enable "auto-upgrade" to upgrade minor versions automatically. 

See `--auto-upgrade {false, true}` at https://learn.microsoft.com/en-us/cli/azure/k8s-extension?view=azure-cli-latest#az-k8s-extension-create

**Comment by macolso** [comment]
@mikkelhegn - are we good to resolve this issue?
